### PR TITLE
[0.63] Add key events to TextWin32 and target prop to ViewWin32 (#6490)

### PR DESCRIPTION
**Pull request by ksiler**
* Merge upstream/master

* Add Keyboard Events to TextWin32

* Change files

* yarn api

* Revert "Merge upstream/master"

This reverts commit 722d5f322344e22428d6a454df8c741697d07336.

(cherry picked from commit e08b7dcdee2549eac8b1deb810330129653fb825)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6762)

### CODE REVIEW

File: packages/react-native-win32/src/Libraries/Components/Text/TextWin32.Props.ts [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 type Omit<T, K> = Pick<T, Exclude<keyof T, keyof K>>;
 
-export type TextWin32OmitTypes = RN.TextPropsAndroid &
+export type TextWin32OmitTypes = Omit<RN.TextPropsAndroid, SharedTextPropsAndroidandWin32> &
   RN.TextPropsIOS &
   RN.AccessibilityPropsAndroid &
   Omit<RN.AccessibilityPropsIOS, SharedAccessibilityPropsIOSandWin32> &
   OmittedAccessibilityPropsWin32;
 
+export type SharedTextPropsAndroidandWin32 = {
```
Comment (dannyvv) [inline packages/react-native-win32/src/Libraries/Components/Text/TextWin32.Props.ts:13]: Just checking if intentional that it is `and` and not `And`.

**Review by dannyvv — APPROVED** [review]

File: change/@office-iss-react-native-win32-0323552c-7d62-481b-8e21-862eee63762b.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
```
[the remark this answers is not in the document]
Comment (ksiler) [reply change/@office-iss-react-native-win32-0323552c-7d62-481b-8e21-862eee63762b.json:1]: Thanks, definitely removed that by accident!

File: packages/react-native-win32/src/Libraries/Components/Text/TextWin32.Props.ts (hunk 1 above)
Comment (ksiler) [reply packages/react-native-win32/src/Libraries/Components/Text/TextWin32.Props.ts:13]: I had written it this way to match `SharedAccessibilityPropsIOSandWin32` but it looks off so I updated it to `And`

Comment (ksiler) [reply packages/react-native-win32/src/Libraries/Components/Text/TextWin32.Props.ts:13]: I actually decided to change it back so it remains consistent with what's in master and 0.64

**Review by Khalef1 — APPROVED** [review]

**Pull request merged**